cloud-functions/opencv_background/main.py

```diff
@@ -0,0 +1,151 @@
+import cv2
+import numpy as np
+
+def image_circle_detection():
+    
+    img = cv2.imread('/home/gabe/Documents/example_gauge.jpg')          #Image source
```

Comment:
Can you update the function to accept an image path`image_circle_detection(input_img):`

Then update line 6:
img = cv2.imread(input_img)